pinot-common/src/test/java/org/apache/pinot/common/metrics/AbstractMetricsTest.java

```diff
@@ -48,4 +52,79 @@ public void testAddOrUpdateGauge() {
     controllerMetrics.removeGauge(metricName);
     Assert.assertTrue(controllerMetrics.getMetricsRegistry().allMetrics().isEmpty());
   }
+
+  @Test
+  public void testInvalidConfig() {
+    PinotConfiguration pinotConfiguration = new PinotConfiguration();
+    pinotConfiguration.setProperty(CONFIG_OF_METRICS_FACTORY_CLASS_NAME,
+        "invalid.metrics.factory.class.name");
+    try {
+      PinotMetricUtils.init(pinotConfiguration);
+      Assert.fail("Expected exception thrown");
+    } catch (Exception e) {
+      Assert.assertTrue(e instanceof RuntimeException, "Expected RuntimeException");
+    }
+  }
```

Comment:
Hmm I'm not entirely sure why this happens
```
Error:  Failures: 
Error:    AbstractMetricsTest.testInvalidConfig:63 Expected exception thrown
```
https://github.com/apache/pinot/actions/runs/10125037774/job/28000076994?pr=13697